### ISSUE TITLE
[Preprocessor] detecting isRoute broken on windows

### ISSUE THREAD

**Issue by pixelmund** [AlecAivazis/houdini#170]
Hey i noticed that if i run a project on windows, the preprocessor is not correctly detecting `isRoute`.

```js
const isRoute =
  config.framework !== "svelte" &&
  !config.static &&
  doc.filename.startsWith(path.join(config.projectRoot, "src", "routes"));

```

The startsWith on windows always fails because there is a mismatch of `/` and `\`

e.g. `C:/Users/user/DEV/src/routes/index.svelte C:\Users\user\DEV\src\routes`

If i change the code to this it works:
```js
const isRoute =
  config.framework !== "svelte" &&
  !config.static &&
  path
    .resolve(doc.filename)
    .startsWith(path.join(config.projectRoot, "src", "routes"));
```

I don't have the time to open a PR currently.

**Comment by AlecAivazis** [comment]
Awesome! Thanks for providing the fix too 👍 I'll sneak it into #171

**Comment by pixelmund** [comment]
Hey i think this is still broken.. The fix never made it into that PR as i can tell. Was playing around with a new project and the preprocessor doesn't know if its a route file or component.
